recipes/pyomo/meta.yaml

```diff
@@ -0,0 +1,166 @@
+{% set version = "4.3.11377" %}
+
+package:
+  name: pyomo
+  version: {{ version }}
+
+source:
+  fn: Pyomo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
```

Comment:
I just changed the url as you requested.  We'll see if tests pass ... but I
haven't had luck with this new format.  I think I've seen intermittent
issues with some releases working but others not.

On Wed, Apr 27, 2016 at 3:39 PM, jakirkham <notifications@github.com> wrote:

> In recipes/pyomo/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/449#discussion_r61322616>
> :
>
> > @@ -0,0 +1,166 @@
> > +{% set version = "4.3.11377" %}
> > +
> > +package:
> > +  name: pyomo
> > +  version: {{ version }}
> > +
> > +source:
> > +  fn: Pyomo-{{ version }}.tar.gz
> > +  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
>
> So, they really messed up PyPI. It took a bit of digging and help from
> other people to figure out the cause. Basically because the old PyPI will
> be replaced with Warehouse, we have to deal and change our URLs.
>
> Basically, change the URL here to this
> https://pypi.io/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/449/files/4cc86b46fbc90cac983bfd000d87409c48e32760#r61322616>
>